app/suggestion/forms.py

```diff
@@ -1,22 +1,37 @@
 from flask.ext.wtf import Form
-from wtforms.fields import SubmitField, TextField
+from wtforms.fields import FloatField, StringField, SubmitField
 from wtforms.validators import Email, InputRequired, Length
 
 
 class SuggestionForm(Form):
-    suggestion_text = TextField(
-        'Suggestion',
+    name = StringField(
+        'Name',
+        validators=[InputRequired(), Length(1, 512)]
+    )
+    address = StringField(
+        'Address', validators=[InputRequired(), Length(1, 512)]
+    )
+    latitude = FloatField(
+        'Latitude',
         validators=[InputRequired()]
     )
-    contact_name = TextField(
+    longitude = FloatField(
+        'Longitude',
+        validators=[InputRequired()]
+    )
+    notes = StringField(
+        'Notes',
```

Comment:
change to "Is there anything else about this resource you would like to share?" - http://www.asylumconnect.org/submit-a-resource/